reproducibility/pkg.yaml

```diff
@@ -55,6 +55,20 @@ steps:
       [ -L /usr/lib64 ] && [ -d /usr/lib64 ]
       [ -L /sbin ] && [ -d /sbin ]
       [ -L /usr/sbin ] && [ -d /usr/sbin ]
+    - |
+      # Test for extra files/directories
+      # go and pkg come from base
+      ALLOWED_DIRS="usr etc bin sbin lib lib64 dev proc sys opt run var root tmp home go pkg"
```

Comment:
Err, we perhaps have /pkg in all the copy stages even when they don't have patches, let me see. Should we maybe also remove /pkg from tools somehow?

Reply:
`/pkg` shouldn;'t end up in `/rootfs` unless there was an explicit copy